tests/publiccloud/transfer_repos.pm

```diff
@@ -57,7 +57,17 @@ sub run {
         # Mitigate occasional CSP network problems (especially one CSP is prone to those issues!)
         # Delay of 2 minutes between the tries to give their network some time to recover after a failure
         # For rsync the ~/repos/./ means that the --relative will take efect after.
-        script_retry("rsync --timeout=$timeout -uahRd -e ssh --files-from /tmp/transfer_repos.txt ~/repos/./ '$remote:/tmp/repos/'", timeout => $timeout + 10, retry => 3, delay => 120);
+        # * The --relative (-R) option is implied when --files-from is specified.
+        # * The --dirs (-d) option is implied whn --files-from is specified.
+        # * The --archive (-a) option's behavior does not imply --recursive (-r) when --files-from is specified.
+        # --recursive (-r), --update (-u), --archive (-a), --human-readable (-h), --rsh (-e)
+        script_retry("rsync --timeout=$timeout -ruahd -e ssh --files-from /tmp/transfer_repos.txt ~/repos/./ '$remote:/tmp/repos/'", timeout => $timeout + 10, retry => 3, delay => 120);
+
+        my $total_size = $args->{my_instance}->ssh_script_output(cmd => 'du -hs /tmp/repos');
+        record_info("Repo size", "Total repositories size: $total_size");
+        $args->{my_instance}->ssh_assert_script_run("find ./ -name '*.rpm' -exec du -h '{}' \\; | sort -h > /tmp/rpm_list.txt", timeout => 60);
```

Comment:
```suggestion
        $args->{my_instance}->ssh_assert_script_run("find ./ -name '*.rpm' -exec du -h '{}' + | sort -h > /tmp/rpm_list.txt", timeout => 60);
```

Using `+` instead of `\;` with `find` will only create one process of the command (in this case `du`) for all found files instead of one process per file.